.bazelrc

```diff
@@ -19,24 +19,24 @@ common:local --extra_execution_platforms=@buildbuddy_toolchain//:platform
 common:dev --bes_results_url=https://buildbuddy.buildbuddy.dev/invocation/
 common:dev --bes_backend=grpcs://buildbuddy.buildbuddy.dev
 
+# Common flags to be used with remote cache
+common:cache-shared --remote_upload_local_results
+common:cache-shared --remote_cache_compression
```

Comment:
Should we just enable this and the flag below unconditionally?

Reply:
We are currently using `--noremote_upload_local_results` and no remote cache by default, if no config is provided. So that could be a bigger switch.